server/routes/configurations.js

```diff
@@ -69,4 +72,11 @@ router
     ConfigurationsController.update
   )
 
+router
+  .route(v1Routes.activeConfigs.index)
+  .get(
+    validateRequest(allSchema),
+    ensureUser,
+    ActiveConfigurationsController.index
+  )
```

Comment:
this could also be a query param to the existing endpoint for getting all configs. We're still working with the same entity, configurations, but we want a filtered set of them.

Reply:
👍  [a514e5d](https://github.com/AMP-SCZ/dpdash/pull/739/commits/a514e5d6e2094f848b5234efcacafad10812b1cc)